.github/workflows/publish.yml

```diff
@@ -162,3 +162,10 @@ jobs:
       - name: Push manifest
         run: |
           make -e V=1 LINUXKIT_PKG_TARGET=manifest pkgs
+
+  trigger_assets:
+    if: ${{ (startsWith(github.ref, 'refs/tags/')) && (github.event.repository.full_name == 'lf-edge/eve') }}
+    needs: [manifest, verification, eve]
+    uses: lf-edge/eve/.github/workflows/assets.yml@master
```

Comment:
I think it should use assets.yml from 9.4 tag release. Changing things in master could break backward compatibility for this or am I missing something?

Reply:
If we refer the actions for `[11.8.0](https://github.com/lf-edge/eve/tree/11.8.0)` tag. the assets trigger was pointed to the the master branch and the trigger was successful.

<img width="1769" alt="image" src="https://github.com/lf-edge/eve/assets/126680600/1b76e04a-c895-49e0-a062-eb03216c7135">

Reply:
What if we change the way assets are build? For instance, in master we add new UEFI file or anything actually and then workflow from 9.4 (which doesn't have this) will trigger master and will fail. Why not use local assets.yml file which is in this branch ?

Reply:
I would prefer to keep it simple For eg. in the tag the first two digits separated by `.` will always be the branch name  . We can then append the `-stable` so that it will be standard across all the `lts` releases.

Reply:
What I'm saying is we do 
`uses: .github/workflows/assets.yml@master`
instead of
`lf-edge/eve/.github/workflows/assets.yml@master`

So that we trigger assets.yml file which is on this specific branch and not on EVE master. That way we don't have to maintain compatibility of assets.yml across all of EVE versions

Reply:
I think using a local (not a master) action makes sense, as it keeps the action in sync with the way we build the package.

Reply:
@uncleDecart this makes sense. Nor sure why I added the reference `@master` :)

Reply:
@OhmSpectator I would also prefer the same. Changes are done. Once this is working as expected, I will update it for 10 and 11 stable branch.